### DISCUSSION TITLE
Airflow Keycloak Intregation

### DISCUSSION THREAD

**Discussion by nitinarora27** [apache/airflow#18581] [General]
Hi Team,

We are trying to configure the keycloak with Airflow 2.x through webserver_config.py and followed many articles but i did not get the success to configure it.

Can you please provide the document through i can configure the keycloak with Airflow2.x .

Advance Thanks for your help !

**Comment by potiuk** [comment]
I do not think there is ready-to-use-document for that. But if you get some, contributing it back to Airflow documentation would be cool

**Comment by koskoskos** [comment]
> Hi Team,
> 
> We are trying to configure the keycloak with Airflow 2.x through webserver_config.py and followed many articles but i did not get the success to configure it.
> 
> Can you please provide the document through i can configure the keycloak with Airflow2.x .
> 
> Advance Thanks for your help !

Hello!
Any luck with this task? I've read multiple articles, tried a tons of helm values variants, no success at all.
